### PR TITLE
fix(usm): add aria-label to tooltip close button

### PR DESCRIPTION
**Pull request by QingyuChai**
Adding the aria-label directly to the tooltip requires updating at least 20 tests, and 150+ snapshots. Trevor recommended the idea of a separate component for the close button for the tooltip, which seems to work! The existing BUIE close button won't work, due to a mismatched icon size, as well as no support for localization without a wrapper.

image

### CODE REVIEW

File: src/components/tooltip/__tests__/__snapshots__/Tooltip.test.tsx.snap [hunk 1]
```diff
@@ -247,16 +247,9 @@ exports[`components/tooltip/Tooltip render() should render with close button if
     role="tooltip"
   >
     hi
-    <PlainButton
-      className="tooltip-close-button"
+    <CloseButton
```
Comment (benjamin-shen) [inline src/components/tooltip/__tests__/__snapshots__/Tooltip.test.tsx.snap:250]: Should we add a new snapshot test for this component?

**Review by benjamin-shen — COMMENTED** [review]
LGTM, left one comment

File: src/components/tooltip/__tests__/__snapshots__/Tooltip.test.tsx.snap (hunk 1 above)
Comment (QingyuChai) [reply src/components/tooltip/__tests__/__snapshots__/Tooltip.test.tsx.snap:250]: think we're leaning away from snapshots for new tests, so I'll add some manual tests! good ide atho

**Review by benjamin-shen — APPROVED** [review]
Nice :)

File: src/components/tooltip/CloseButton.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+// @ts-ignore flow import
+import messages from '../../common/messages';
+
+type Props = {
+    intl: IntlShape;
+    onClick: (event: React.SyntheticEvent<HTMLButtonElement, Event>) => void;
+};
+
+const CloseButton = ({ intl, onClick }: Props) => (
```
Comment (tjuanitas) [inline src/components/tooltip/CloseButton.tsx:15]: it's out of the scope of the original issue but in the future we should look into if we could combine these two components: https://github.com/box/box-ui-elements/tree/master/src/components/close-button

**Review by tjuanitas — APPROVED** [review]

**Pull request merged**